template/Makefile

```
@@ -14,6 +14,13 @@ install:
 dev: update-box
 	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} ANSIBLE_ARGS='--skip-tags "test"' vagrant up --provision
 
+linter:
+ifneq ('$(shell docker images -q github/super-linter)','')
+	docker run -e RUN_LOCAL=true -v "${PWD}:/tmp/lint/" github/super-linter
+else
+	@echo "Can not find 'github/super-linter'. You need to 'docker pull github/super-linter:latest'"
+endif
```

I think this block is not necessary, due to if the image not found on the local machine it will be automatically downloaded from the public Docker registry (dockerhub)
```suggestion
	docker run -e RUN_LOCAL=true -v "${PWD}:/tmp/lint/" github/super-linter
```

---

It didn't happend when I tryed to to run `make linter`. But that might be because i wasn't logged in? :thinking:

---

Maybe then
```suggestion
	docker pull github/super-linter
	docker run -e RUN_LOCAL=true -v "${PWD}:/tmp/lint/" github/super-linter
```

---

No, you've right! I deleted the image and run `docker run -e RUN_LOCAL=true -v "${PWD}:/tmp/lint/" github/super-linter` after and it downloded the image automatically. Must be caused by me not being logged in. 

I'll commit your first suggestion 👍